jetson/db/db_connection.py

```diff
@@ -0,0 +1,125 @@
+import mysql.connector
+import config
+from contextlib import contextmanager, closing
+
+
+class Table:
+    def __init__(self):
+        pass
+
+
+class IMAGE(Table):
+    """Class defines the Image table in the database with init inputs being the columns in the table.

```

Comment:
Can you clean up this description.  Language like "class defines" and "init inputs" are kind of redundant here